tests/inputFields.spec.ts

```diff
@@ -5,91 +5,84 @@ test.beforeEach( async({page}) => {
   //1. Select the PET TYPES menu item in the navigation bar
   await page.getByText('Pet Types').click()
   //2. Add assertion of the "Pet Types" text displayed above the table with the list of pet types
-  expect(await page.locator('h2').getByText('Pet Types').isVisible()).toBeTruthy()
+  expect(await page.locator('h2').getByText('Pet Types').textContent()).toEqual('Pet Types')
 })
 
 test('Update pet type', async ({page}) => {
-  test.slow()
   //3. Click on "Edit" button for the "cat" pet type
   //assumes cat is always first
   await page.getByText('Edit').first().click()
 
   //4. Add assertion of the "Edit Pet Type" text displayed
-  expect(await page.getByText('Edit Pet Type').isVisible()).toBeTruthy()
+  expect(await page.getByText('Edit Pet Type').textContent()).toEqual('Edit Pet Type')
```

Comment:
Better.
But how can you make this assertion even more compact using Locator Assertion?